Source/Core/DiscIO/DirectoryBlob.h

```diff
@@ -196,8 +200,7 @@ class DirectoryBlobPartition
       const std::function<void(std::vector<FSTBuilderNode>* fst_nodes, FSTBuilderNode* dol_node)>&
           fst_callback);
 
-  // We do not allow copying, because it might mess up the pointers inside DiscContents
-  DirectoryBlobPartition(const DirectoryBlobPartition&) = delete;
+  DirectoryBlobPartition(const DirectoryBlobPartition&);
   DirectoryBlobPartition& operator=(const DirectoryBlobPartition&) = delete;
```

Comment:
If you've fixed copies being unsafe, you should be able to remove this line too.